### ISSUE TITLE
Warn about capturing empty output; var=$(cmd &>/dev/null)

### ISSUE THREAD

**Issue by knirch** [koalaman/shellcheck#408]
Noticed in a refactoring branch that the behaviour for SC2069 had changed so I didn't need disable statements surrounding my valid uses anymore, which got me thinking. Wouldn't a checker for code like this make sense; 

```bash
foo=$(cmd >/dev/null)
foo=$(cmd >/dev/null 2>&1)
foo=$(cmd &>/dev/null)
cmd >/dev/null | cmd
cmd >/dev/null 2>&1
cmd &>/dev/null
# etc..
```
